docs/source/extensions/strategy.rst

```diff
@@ -0,0 +1,111 @@
+.. _strategy:
+
+########
+Strategy
+########
+
+Strategy depicts the training strategy to be used by the :doc:`Lightning Trainer <../common/trainer>`. It can be controlled by passing different
```

Comment:
Strategy depicts training strategy is circular.

We should define what strategy is meant to encompass, and only afterward list out how someone can use it. For instance, Strategy controls the model distribution across training, evaluation, and prediction.

Reply:
I would agree, strategy does not fully fit into what extensions are. it is more an Option, as in, many different strategies are available to choose from but it is required to have one.

Reply:
Strategies can also be "extended", which is probably why @kaushikb11 has put it here, but this is definitely a different category than Callbacks for example.